src/frontend/web_application/src/components/Modal/index.jsx

```diff
@@ -9,6 +9,7 @@ const Modal = ({ className, title, children, onClose, ...props }) => (
   <ReactModal
     className={classnames('m-modal', className)}
     overlayClassName="m-modal__overlay"
+    appElement={document.body}
```

Comment:
I think using `document` can be a problem when rendering server side, using a css selector should be better. 
And  add a comment to explain why it is important (useful to limit the screenreader to the content of modal)

Reply:
ok, but which selector then?

Reply:
`"#root"` should be the selector to use I suppose.

https://github.com/CaliOpen/Caliopen/blob/master/src/frontend/web_application/src/index.jsx#L43